app/pages/sections/ArrivalSection.scala

```diff
@@ -16,12 +16,11 @@
 
 package pages.sections
 
-import play.api.libs.json.JsPath
-import pages.QuestionPage
+import play.api.libs.json.{JsObject, JsPath}
 
-case object IncidentSection extends QuestionPage[Nothing] {
+case object ArrivalSection extends Section[JsObject] {
```

Comment:
Interesting... We don't have this in departures. I think we can do without this nesting but I'm now second guessing myself 😄

Reply:
The whole thing is the arrival is my thinking. I think let's keep it consistent with departures if you guys agree

Reply:
Ok cool, will remove